SPEC.md

```diff
@@ -7367,6 +7368,93 @@ Example output:
 </p>
 </details>
 
+### ✨ `join_paths`
+
+```
+File join_paths(File|String, String)
```

Comment:
This probably depends on the rules of casting between `File` and `Directory` but should the definitions be

```
File join_paths(Directory|String, String)
```
etc

Reply:
We haven't added `Directory` support to 1.2 yet :)